dartagnan/src/main/java/com/dat3m/dartagnan/GlobalSettings.java

```diff
@@ -38,6 +38,10 @@ public static String getCatDirectory() {
         return env + "/cat";
     }
 
+    public static String getBoundsFile() {
+        return getOutputDirectory() + "/bounds.csv";
+    }
+
```

Comment:
When doing unit tests, thsi should probably return the empty string.
Btw., why do we even need a default location for saving bounds anyways?

Reply:
Initially I was not planning to implement an option to allow the specify the saving path, but one of my colleagues requested this. 

You are right that we don't need this anymore.